packages/cli/src/credentials/K8sCredentialManager.ts

```diff
@@ -25,11 +30,18 @@ class K8sCredentialManager extends AbstractCredentialManager {
     private kubeConfig: KubeConfig;
     private kc: any;
 
-    constructor(service: string, displayName: string = "K8s Credential Manager") {
+    constructor(
+        service: string,
+        displayName: string = "K8s Credential Manager"
+    ) {
         super(service, displayName);
         this.allServices = [service || K8sCredentialManager.SVC_NAME];
         if (this.defaultService === K8sCredentialManager.SVC_NAME) {
-            this.allServices.push("@zowe/cli", "Zowe-Plugin", "Broadcom-Plugin");
+            this.allServices.push(
+                "@zowe/cli",
+                "Zowe-Plugin",
+                "Broadcom-Plugin"
```

Comment:
Curious if we should drop some services?
I believe that a long time ago there was an effort to migrate users behind the scenes.
Meaning, the next time a credential was touched, we would store it in the newest service.

It probably doesn't hurt to have them there though.

Reply:
Wondering if we should drop the `allServices` definition altogether. I know the SCS plugin needed it for backwards compatibility, but `AbstractCredentialManager` doesn't require it and it's not needed anywhere for the K8s plugin as far as I can tell.